### ISSUE TITLE
Improve report text for core conformance classes

### ISSUE THREAD

**Issue by bermud** [opengeospatial/teamengine#280]
Related to this result:

```
Test Name: wfs20
Test version: 1.27
Time: 2017-12-04T19:38:58.768Z
Basic Conformance Classes:
   * Preconditions
   * All GML application schemas
   * GML application schemas defining features
   * Simple WFS

Test INPUT:
   * acceptMediaType : application/zip
   * wfs : http://wfst.axl.aero/AxlRest/wfs?service=WFS&version=2.0.0&request=GetCapabilities
   * xsd : http://wfst.axl.aero/AxlRest/wfs?service=WFS&version=2.0.0&request=DescribeFeatureType
Result: 
   * Number of conformance classes tested: 13
   * Passed core (Can be certified): Yes
   * Number of conformance class passed: 10
   * Number of conformance class failed: 0
```
Suggest to do the following:

- [ ] Rename the text **Basic Conformance Classes** to **Core Conformance Classes (Implementation passing these classes can be certified)**
- [ ] **Passed core (Can be certified): Yes**  should appear first after **Result:**

**Comment by keshav-nangare** [comment]
Updated the HTML report as per the suggestion.

![image](https://user-images.githubusercontent.com/14732650/33606411-652566cc-d9e3-11e7-9280-4462725eebc3.png)

**Comment by keshav-nangare** [comment]
Fixed with this commit: b32ccc70226c72ba0753c7eb11875d64c3a8de3c
